monocle/worker.py

```diff
@@ -897,6 +921,32 @@ def smart_throttle(self, requests=1):
         except (TypeError, KeyError):
             return False
 
+    async def add_lure_pokestop(self, pokestop):
```

Comment:
@evenly-epic-mule is right.  At a minimum, split this into 2 PRs.  Adding lures takes away money and encourages botting.  Monocle community is not about that.  Some people may want that feature and may implement it themselves with their own experience.  

_Sent from my LGE LG-H820 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_